Create new token description (#282)
sora-xor/sora-docs: src/karma.md

---
title: Chameleon KARMA | SORA Docs
head:
  - - meta
    - name: description

>       content: Chameleon pool, liquidity providers in the XOR-ETH pool, buy-back and burn mechanizm, distributed to the Demeter farm, deflationary mechanism

Reviewer: Here you would add a summary of the content of the page, not just the keywords

>       content: Chameleon pool, liquidity providers in the XOR-ETH pool, buy-back and burn mechanizm, distributed to the Demeter farm, deflationary mechanism

Reviewer: ```suggestion
      content: The KARMA token provides farming rewards to liquidity providers in the XOR-ETH pool on Polkaswap. It is specifically designed to incentivize liquidity providers and support the stability of the Chameleon pools on Polkaswap.
```

  - - meta
    - name: keywords
      content: token burn, Chameleon, XOR, ETH, Demeter, KXOR
---

# Chameleon (KARMA)

> KARMA is a key token in the SORA ecosystem, specifically designed to incentivize liquidity providers and support the stability of the Chameleon pools on Polkaswap. Here's a detailed look at how KARMA works.

Reviewer: ```suggestion
KARMA is a key SORA ecosystem token, specifically designed to incentivize liquidity providers and support the stability of the Chameleon pools on Polkaswap. Here's a detailed look at how KARMA works.
```

## Purpose and Incentives

The KARMA token is used to provide farming rewards to liquidity providers in the XOR-ETH pool on Polkaswap.

> By participating in liquidity provision, users are rewarded with KARMA, encouraging more liquidity in the Chameleon pools.

Reviewer: ```suggestion
By participating in liquidity provision for this pool, users are rewarded with KARMA, encouraging more liquidity in the Chameleon pools.
```

## Minting and Burning Mechanism

For each Kensetsu XOR (KXOR) minted from collateralized deposits, an additional 1% of KXOR is minted specifically to buy back and burn KARMA tokens.

> This buy-back-and-burn mechanism helps to regulate the supply of KARMA, supporting its value.

Reviewer: ```suggestion
This buy-back-and-burn mechanism helps regulate the KARMA supply, supporting its value.
```

## Redistribution and Farming Rewards

80% of the KARMA tokens that are bought back and burned will be reminted and distributed to the Demeter farm.

These reminted KARMA tokens are then used as farming rewards for liquidity providers in the Chameleon pools, initially focusing on the XOR-ETH pool on Polkaswap.

## Deflationary Aspect

> KARMA also has a deflationary mechanism. For every 10 million XOR burned, 1 KARMA token is minted.

Reviewer: Will more Karma be minted after the initial distribution? or is this describing the burn drop mechanism?

Author: No, no more Karma will be minted. The users have the opportunity to mint Karma via burn XOR until June 06.

> KARMA also has a deflationary mechanism. For every 10 million XOR burned, 1 KARMA token is minted.

Reviewer: ```suggestion
KARMA also has a deflationary mechanism. During the initial mint, which ended on June 6th, for every 10 million XOR burned, 1 KARMA token was minted.
```

After this initial minting, the KARMA token becomes deflationary, with 1% of minted KXOR used to buy back and burn KARMA tokens.

> ## In summary

Reviewer: Please start the article with the summary
